sklearn/neighbors/_binary_tree.pxi

```diff
@@ -1362,7 +1363,8 @@ cdef class BinaryTree:
     def query_radius(self, X, r, int return_distance=False,
                      int count_only=False, int sort_results=False):
         """
-        query_radius(self, X, r, count_only = False):
+        query_radius(X, r, return_distance=False,
```

Comment:
Nice. For other reviewers, according to [numpydoc](https://numpydoc.readthedocs.io/en/latest/format.html#sections):
> For some functions (notably those written in C) the signature is not available, so we have to specify it as the first line of the docstring